### PR TITLE
Failed to find expected logs while site.xcatdebugmode=1 for ubuntu on ppc64le

### PR DESCRIPTION
**Pull request by penguhyang**
for ubuntu14, change the permission to make the logs can be saved.
for ubuntu15, fix the procedure to make the logs can be saved.

### CODE REVIEW

**Comment by penguhyang** [comment]
fix issue: #954

File: xCAT/postscripts/syslog [hunk 1]
```diff
@@ -152,6 +154,12 @@ config_rsyslog_V8()
     fi
   fi
 
+  # backup rsyslog.conf
+  if [ ! -f $conf_file.XCATORIG ]; then
+    cp -f $conf_file $conf_file.XCATORIG
+  else
+    cat $conf_file > $conf_file.XCATORIG
```
Comment (immarvin) [inline xCAT/postscripts/syslog]: the $conf_file.XCATORIG is used to backup the original syslog configuration on rsyslog installation, do not overwrite it when it has already existed

File: xCAT/postscripts/syslog [hunk 2]
```diff
@@ -169,11 +177,13 @@ config_rsyslog_V8()
...
+                     s/#input(type="imudp" port="514")/input(type="imudp" port="514")/; 
+                     s/#module(load="imtcp")/module(load="imtcp")/; 
+                     s/#input(type="imtcp" port="514")/input(type="imtcp" port="514")/; 
+                     s/\$PrivDropToGroup syslog/\$PrivDropToGroup adm/' $conf_file
       chown syslog:adm /var/log/messages
-    fi
-    # backup rsyslog.conf
-    if [ ! -f $conf_file.XCATORIG ]; then
-      cp -f $conf_file $conf_file.XCATORIG
+      chown syslog:adm /var/log/xcat
```
Comment (immarvin) [inline xCAT/postscripts/syslog]: whether "-R" option is needed?
````
       -R, --recursive
              operate on files and directories recursively
````
if the log files are created after the postscript "syslog" is run, will the files inherit the owner and group of the directory?if not, maybe we need to create them on xCAT installation

File: xCAT/postscripts/syslog (hunk 1 above)
Comment (penguhyang) [inline xCAT/postscripts/syslog]: fixed.

File: xCAT/postscripts/syslog (hunk 2 above)
Comment (neo954) [inline xCAT/postscripts/syslog]: ``-R`` is not needed.

Line 174 and line 180 is no use actually. ``rsyslog`` will create the log file as needed. No need to ``touch`` it first at all.

File: xCAT/postscripts/syslog [hunk 3]
```diff
@@ -213,6 +219,11 @@ config_rsyslog_V8()
     # add xCAT entries
     echo "# xCAT settings" >> $remoteconf
     echo "*.* @$master" >> $remoteconf
+
+    if ( pmatch $OSVER "ubuntu*" ) || ( is_lsb_ubuntu ) || ( pmatch $OSVER "debian*" ); then
+        echo "# xCAT settings" >> $conf_file
+        echo "*.* @$master" >> $conf_file
+    fi
```
Comment (neo954) [inline xCAT/postscripts/syslog]: ``$remoteconf`` is already used, why do the same thing again in the main configuration file ``$conf_file``.

Comment (penguhyang) [inline xCAT/postscripts/syslog]: fixed

**Pull request merged**